### DISCUSSION TITLE
Discord Link?

### DISCUSSION THREAD

**Discussion by CreateShiro** [PlayCover/PlayCover#592] [Q&A] [answered]
The discord link in is expired.  Can someone share the new link or update the one in https://docs.playcover.io/

**Comment by ZhichGaming** [comment, marked as answer]
Thank you for pointing that out, I have just submitted a PR here: https://github.com/PlayCover/PlayBook/pull/43

You can use the link in that PR or just the vanity URL discord.gg/playcover.
